pkg/scheduler/actions/shuffle/shuffle_test.go

```diff
@@ -0,0 +1,331 @@
+/*
+ Copyright 2022 The Volcano Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package shuffle
+
+import (
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	schedulingv1 "k8s.io/api/scheduling/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+
+	schedulingv1beta1 "volcano.sh/apis/pkg/apis/scheduling/v1beta1"
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/cache"
+	"volcano.sh/volcano/pkg/scheduler/conf"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+	"volcano.sh/volcano/pkg/scheduler/plugins/rescheduling"
```

Comment:
shuffle action should not dependent on rescheduling plugin.

Reply:
Just as `allocate_test.go` and `preempt_test.go` works, all the UT of actions should work with some registered plugins and then get an expected output. So this UT registers `rescheduling` plugin.

Reply:
Let's build a mock plugin for action UT, and we should also update allocate_test & preempt_test to have mock plugins.

Reply:
It is reasonable in some degree. Perhaps we can submit another PR especially for that.

Reply:
> It is reasonable in some degree. Perhaps we can submit another PR especially for that.

How to handle CI error?